src/api/view/message.py

```diff
@@ -0,0 +1,11 @@
+from flask import jsonify
```

Comment:
I see this import jsonify in a lot of the files, what from json is it importing?

Reply:
So jsonify is a function provided by Flask that transforms the RealDictRows returned by PostgreSQL into JSON.